tests/sentry/integrations/slack/test_disable.py

```diff
@@ -76,11 +76,16 @@ def test_fatal_and_disable_integration(self):
     def test_email(self):
         client = SlackClient(integration_id=self.integration.id)
         with self.tasks():
-            notify_disable(self.organization, self.integration.provider, client._get_redis_key())
+            notify_disable(self.organization, self.integration, client._get_redis_key())
         assert len(mail.outbox) == 1
         msg = mail.outbox[0]
         assert msg.subject == "Action required: re-authenticate or fix your Slack integration"
-        assert (f"/settings/integrations/{self.integration.provider}") in msg.body
+        assert (
+            self.organization.absolute_url(
+                f"/settings/{self.organization.slug}/integrations/{self.integration.provider}"
+            )
+            in msg.body
+        )
```

Comment:
Could you add a test to make sure the URL is correct for sentry apps? You can add it in https://github.com/getsentry/sentry/pull/53522 after this is merged and you rebase